CHANGELOG.md

```diff
@@ -30,6 +31,7 @@ v1.15.0 (2017-09-15)
     received over the wire.
 
 
+
```

Comment:
woah man, that's one too many

![sed3azx](https://user-images.githubusercontent.com/613317/30607692-c02d0e44-9d2a-11e7-93c7-f8d235386114.gif)

Reply:
Fixed